### ISSUE TITLE
Cannot find module '*.ts' from "*.cjs.development.js" after publish and importing it

### ISSUE THREAD

**Issue by PabloSzx** [palmerhq/tsdx#94]
### Current Behavior

```bash
yarn publish library_name
```

```javascript
import "library_name"
// Cannot find module '*.ts' from "library_name.cjs.development.js"
```

### Expected behavior

Being able to use the code generated from the build.

### Additional context

> A simple example

https://github.com/PabloSzx/crypto-use/tree/master/src

### Suggested solution(s)

Build all the modules in "src" into native javascript and transform ".ts" to ".js"

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX          | 0.5.8 
| TypeScript       | 3.4.5
| Browser          | not relevant
| npm/Yarn         |  yarn@1.15.2 npm@6.9.0
| Operating System | any

**Comment by jaredpalmer** [comment]
I checked out your code and did a build. I was not able to reproduce this. Can you try 0.5.9?

**Comment by PabloSzx** [comment]
0.5.9 appareantly fixes some of the issues, but now i'm having this issue with lodash as dependency

```javascript
Cannot find module 'lodash-es' from 'crypto-use.cjs.development.js'
```

**Comment by jaredpalmer** [comment]
Did you install both lodash and lodash-es?

**Comment by jaredpalmer** [comment]
@PabloSzx you need to install both lodash and lodash-es. @TrySound is their a way we can sniff out if a user has only installed lodash and throw an error?

**Comment by PabloSzx** [comment]
when using [microbundle](https://github.com/developit/microbundle) i don't need to install *lodash-es*

**Comment by PabloSzx** [comment]
and now jest is having issues

```javascript
Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    C:\Users\PabloSz\Documents\crypto-use\node_modules\lodash-es\lodash.js:10
    export { default as add } from './add.js';
    ^^^^^^

    SyntaxError: Unexpected token export

       5 | var crypto = _interopDefault(require('crypto-js'));
       6 | var rabbit = _interopDefault(require('crypto-js/rabbit'));
    >  7 | var _ = require('lodash-es');
         |         ^
       8 | var ___default = _interopDefault(_);
       9 | var serialize = _interopDefault(require('serialize-javascript'));
      10 | var mongodb = require('mongodb');

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:471:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:513:25)
      at Object.<anonymous> (dist/crypto-use.cjs.development.js:7:9)
```

**Comment by TrySound** [comment]
@jaredpalmer Not sure. I would make this optimisation optional for those who understand it.

**Comment by PabloSzx** [comment]
It's not only a jest problem, it's in the building itself, i published the build from tsdx, and "tested" it manually on a node code, and it gives the same error.

```javascript
λ node index.js
C:\Users\PabloSz\Documents\tsdxtest\node_modules\lodash-es\lodash.js:10
export { default as add } from './add.js';
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:743:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (C:\Users\PabloSz\Documents\tsdxtest\node_modules\crypto-use\dist\crypto-use.cjs.development.js:7:9)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
```

**Comment by PabloSzx** [comment]
Bug fixed with v0.5.10 commit af526a5fc672694bdef543da036af890427568a9
